src/contest.c

```diff
@@ -998,6 +998,8 @@ void ResetLinkContestBoolean(void)
 
 static void SetupContestGpuRegs(void)
 {
+    u16 savedIme;
```

Comment:
This variable is not used at all and the build matches with this gone. Since the last travis build failed anyway, why not add this
```suggestion
```